### PR TITLE
Send screenshot when stopping VNC

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Hold | [Ticket](https://pi-top.atlassian.net/browse/FUR-2495) |

#### Main changes
-

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,8 +22,9 @@ RUN git clone https://github.com/pi-top/pi-topOS-Apt-Source.git && \
     rm -rf /pi-topOS-Apt-Source
 
 # Install pt-web-vnc
+COPY pt-web-vnc_0.3.1-3~8.gbpfbf64b_all.deb /tmp/
 RUN apt-get update && \
-    apt-get install -y pt-web-vnc && \
+    apt-get install -y /tmp/pt-web-vnc_0.3.1-3~8.gbpfbf64b_all.deb && \
```
Comment (angusjfw) [inline Dockerfile:27]: pt-web-vnc should be released and this changed back. then really should do a final test but otherwise looks good

**Comment by angusjfw** [comment]
Looks good to me (not approving as it's a draft and the pt-web-vnc dependency needs updating).

I spent a while double checking this hasn't made the issue of not sending output from short running processes worse, since we thought there might be an issue there when we tested before. I don't think this has changed that, it's as before (shell echo hi doesn't usually work, python print('hi') always should as it's a lot slower).

**Review by angusjfw — APPROVED** [review]
